other/install_2-1_postgresql.sql

```diff
@@ -1218,9 +1218,8 @@ CREATE TABLE {$db_prefix}messages (
 # Indexes for table `messages`
 #
 
-CREATE UNIQUE INDEX {$db_prefix}messages_id_board ON {$db_prefix}messages (id_board, id_msg);
+CREATE UNIQUE INDEX {$db_prefix}messages_id_board ON {$db_prefix}messages (id_board, id_msg, approved);
```

Comment:
This is a unique Index, by adding this col, you change his behavior.

Reply:
This index is unique already, as id_msg is unique on the message table.

You are correct in that this index doesn't really need to be UNIQUE...  E.g., it is not helping us enforce a true constraint.  It is primarily helping us get to messages on a board.